packages/api-server/src/methods/modules/gw.ts

```diff
@@ -1,11 +1,14 @@
-import { parseGwRpcError } from "../gw-error";
-import { RPC } from "@godwoken-web3/godwoken";
+import { parseGwRpcError, parseGwRunResultError } from "../gw-error";
+import { RPC, RunResult } from "@godwoken-web3/godwoken";
 import { middleware } from "../validator";
-import { HexNumber } from "@ckb-lumos/base";
+import { HexNumber, HexString } from "@ckb-lumos/base";
 import { Store } from "../../cache/store";
 import { envConfig } from "../../base/env-config";
 import { CACHE_EXPIRED_TIME_MILSECS, GW_RPC_KEY } from "../../cache/constant";
 import { logger } from "../../base/logger";
+import { DataCacheConstructor, RedisDataCache } from "../../cache/data";
+
+const blake2b = require("blake2b");
```

Comment:
It can replaced by https://github.com/nervosnetwork/lumos/blob/v0.18.0-rc7/packages/base/lib/utils.d.ts#L15
CKB use blake2b with a personalization

Reply:
updated.